routes/configOverview.js

```diff
@@ -1,62 +1,26 @@
 
-var linz = require('../'),
-    async = require('async');
+var linz = require('../');
 
 /* GET /admin/config/:config/overview */
 var route = function (req, res) {
 
-	var locals = {
-		config: req.linz.config,
-		record: req.linz.record,
-		permissions: req.linz.config.linz.formtools.permissions,
-		formtools: req.linz.config.linz.formtools
-	};
+    var locals = {
+        config: req.linz.config,
+        record: req.linz.record,
+        permissions: req.linz.config.linz.formtools.permissions,
+        formtools: req.linz.config.linz.formtools
+    };
 
-	async.series([
+    // Transform overview.body DSL into data object that can be rendered by view
+    linz.formtools.overview.body(req, res, req.linz.record, req.linz.config, req.linz.config.linz.formtools.overview.body, function (err, overviewData) {
 
-        function (cb) {
+        if (!err) {
```

Comment:
I know this probably existed in the old code, but we should be handling the error here. Can you surface it via `res`?